base/bucket_test.go

```diff
@@ -358,44 +370,64 @@ func mockCertificatesAndKeys(t *testing.T) {
 	certBytes, err = x509.CreateCertificate(rand.Reader, &clientTemplate, &rootTemplate, &clientKey.PublicKey, rootKey)
 	assert.NoError(t, err, "Client certificate should be generated")
```

Comment:
This and the errors above I think should be `require.NoError`. The tests won't work if this isn't successful.

Reply:
This is fixed in latest revision.